Cargo.toml

```
@@ -35,6 +35,7 @@ docker_credential = { version = "1.3.1", default-features = false }
 etcetera = { version = "0.10.0", default-features = false }
 figment = { version = "0.10.19", default-features = false, features = ["json", "env"] }
 hyper = { version = "1.6.0", default-features = false }
+indicatif = "0.18.0"
```

```suggestion
indicatif = { version =" 0.18.0", default-features = false }
```

Just a suggestion to keep parity, if you need any features feel free to enable them